public/index.html

```diff
@@ -7,6 +7,6 @@
   <title>Document</title>
 </head>
 <body>
-  <div id="root" class="p-4"></div>
```

Comment:
- [ ] It would be better if we improve our PR, 
```
In this milestone, I implemented the following;
- Add all element in the design
- Style exactly as the given design
- Implement CSS best practices
```
Formatting our PR helps our repo looks professional